### ISSUE TITLE
Deprecated WebSecurityConfigurerAdapter needs to be updated.

### ISSUE THREAD

**Issue by Abhijeetmishr** [jmprathab/MyHome#279]
## ⚠️ Is your feature request related to a problem? Please describe

In the lastest Springboot version WebSecurityConfigurerAdapter is deprecated.

## 💡 Describe the solution you'd like
I wanted to update the existing code for WebSecurityConfigurerAdapter in WebSecurity class to latest one.

## 🤚 Do you want to develop this feature yourself?

- [x] Yes
- [ ] No

**Comment by PratikS17** [comment]
Hi @Abhijeetmishr 
I would like to work on this issue.

My approach : 
1. Remove WebSecurityConfigurerAdapter
2. Export SecurityFilterChain bean
3. Instead of using configure(AuthenticationManagerBuilder), we export DaoAuthenticationProvider bean (child of AuthenticationProvider) and pass it to HttpSecurity’s authenticationProvider() method
4. Export AuthenticationManager bean

Please let me know if my approach is correct or it needs some refactoring. If it sounds good then can you please assign this issue to me.

Thanks,
Pratik

**Comment by Abhijeetmishr** [comment]
@PratikS17 yes you can start but I think this project is not active anymore because no one is replying to the queries.
